lib/models/Dog.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const dogSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  age: {
+    type: Number,
+    required: true,
+    min: 0
+  },
+  color: {
+    type: String,
+    required: true
+  }
+
+});
+
+module.exports = mongoose.model('Dog', dogSchema);
```

Comment:
Include a new line at the end of each file. You can configure VSCode to do this for you automatically.